runbot_travis2docker/models/runbot_branch.py

```diff
@@ -33,6 +34,30 @@ def _compute_name_weblate(self):
                         dict(match.groupdict(), branch=branch['branch_name']))
             branch.name_weblate = name
 
+    @tools.ormcache('ssh')
+    def _ssh_keyscan(self, ssh):
+        """This function execute the command 'ssh-keysan' to avoid the question
+        when the command git fetch is excecuted.
+        The question is like to:
+            'Are you sure you want to continue connecting (yes/no)?'"""
+        cmd = ['ssh-keyscan', '-p']
+        match = re.search(r'@(?P<port_host>[^/]+)', ssh)
+        if match:
+            port_host = match.groupdict()['port_host'].split(':')
```

Comment:
What about use regex to get `host:port`? (instead of split(':')

Reply:
@moylop260 
I don't known how to write a regular expression to avoid the function split.
I need more knowledge in this matter, sorry :disappointed: 
Can you help me with this regular expression ?